r/R/config.R

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#' Return number of threads used by the global CPU thread pool in libarrow
+#'
+#' @export
```

Comment:
If you want these functions to appear on the doc website table of contents, add `cpu_count` to the list in `r/_pkgdown.yml`, probably somewhere around L130. If you don't want to publicize these functions, add a line with `#' @keywords internal` around here (the man pages will still show up on the website, just not included on the ToC). Either way is fine with me.